src/Details.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
 import { Link } from 'react-router-dom'
-export default function Details() {
-    return (
-        <div>Hello, this will be the details page for each Movie & TV show :)
+export default class Details extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            initialMessage: 'Hello, this will be the details page for each Movie & TV show :)'
+        };
+    }
+
+    componentDidMount(){
+        setTimeout(() =>{
+            this.setState({
+                initialMessage: 'Coming soon! :)'
+            });
+        }, 3000);
+    }
+    
+    render() {
+        return (
+            <div>{this.state.initialMessage}
     <br></br>
```

Comment:
Please indent this line. You can do it automatically with right click -> format document

Reply:
Vettel would be proud. Race winning performance